WpfApp5/WpfApp5/RelayCommand.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace WpfApp5
+{
+    public class RelayCommand : ICommand
```

Comment:
Command belongs to ViewModel layer.
Should be placed in some folder, not in root.

Reply:
-1 to Concerns separation.